CMakeLists.txt.in

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.11)
```

Comment:
I think that file can be removed now.

Reply:
do we need to increase the minimum version then?